src/node_snapshotable.h

```diff
@@ -14,6 +14,21 @@ struct EnvSerializeInfo;
 struct SnapshotData;
 class ExternalReferenceRegistry;
 
+using SnapshotIndex = size_t;
+
+struct PropInfo {
+  std::string name;     // name for debugging
+  uint32_t id;          // In the list - in case there are any empty entries
+  SnapshotIndex index;  // In the snapshot
+};
+
+struct RealmSerializeInfo {
```

Comment:
I think this can be in node_realm.h?

Reply:
Yeah. It was added here to avoid circular includes. But it is no longer a problem here.